README.md

```diff
@@ -862,7 +862,9 @@ bar
 If `dotenv-load`, `dotenv-filename` or `dotenv-path` is set, `just` will load
 environment variables from a file.
 
-If `dotenv-path` is set, `just` will look for a file at the given path.
+If `dotenv-path` is set, `just` will look for a file at the given path. It is
+an error if a dotenv file is not found at `dotenv-path`, but not an error if a
+dotenv file is not found with `dotenv-filename`.
```

Comment:
What is the motivation behind the difference in logic?

Reply:
This is just documenting the existing behavior. I'm not sure what the original motivation was.

Reply:
Personally, I find it confusing that `dotenv-path` and `dotenv-filename` have this kind of behavioural difference. I think it would be better if both parameters had the same behaviour.

---

In a more general context, I find the presence of both `dotenv-filename` and `dotenv-path` a bit confusing. Isn't `dotenv-filename` basically a more limited version of `dotenv-path` (ignoring this specific difference in behaviour)? Perhaps it would make sense to deprecate `dotenv-filename` (removing it outright would probably break some justfiles)?